frontend/public/components/pod.jsx

```diff
@@ -133,7 +133,7 @@ const Volume = ({pod, volume}) => {
 };
 
 const ContainerTable = ({heading, containers, pod}) => <div className="co-m-pane__body">
-  <h1 className="co-section-title">{heading}</h1>
+  <h2 className="co-m-pane__heading co-section-heading">{heading}</h2>
```

Comment:
Is `.co-section-heading` compatible with [.co-m-pane__heading--*](https://github.com/sg00dwin/console/blob/e0940475dc5fe5c9aeebb907a35b3658e1b564f2/frontend/public/style/_common.scss#L72-L78)?  I'm not arguing it should be, but the options exist, so it's possible they could be used together...